### PR TITLE
Feature/itinerary vc

### PR DESCRIPTION
**Pull request by DavidWrightOS**
# Itinerary View Controller Setup

Setup the favorites tab UI to display a list of destinations that the user has favorited.

Trello URL: **_https://trello.com/c/1NV5OlRk_**

PR Branch: **_https://github.com/Lambda-School-Labs/LabsPT13-Resfeber-A-IOS/tree/feature/itinerary-vc_**

### CODE REVIEW

**Review by joshrutkowski — APPROVED** [review]
Update looks great!

File: Resfeber/View Controllers/FavoritesViewController.swift [hunk 1]
```diff
@@ -9,22 +9,90 @@
...
+    }
+}
+
+// MARK: - Collection View Delegate
+
+extension FavoritesViewController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard indexPath.row < DestinationData.favoriteDestinations.count else { return }
+        let destination = DestinationData.favoriteDestinations[indexPath.row]
+        print("DEBUG: Tapped destination: \(destination.name)..")
```
Comment (johnnydhicks) [inline Resfeber/View Controllers/FavoritesViewController.swift:96]: Don't forget to remove this too

File: Resfeber/View Controllers/ItineraryViewController.swift [hunk 2]
```diff
@@ -10,22 +10,89 @@ import UIKit
...
+    }
+}
+
+// MARK: - Collection View Delegate
+
+extension ItineraryViewController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard indexPath.row < DestinationData.itineraryDestinations.count else { return }
+        let destination = DestinationData.itineraryDestinations[indexPath.row]
+        print("DEBUG: Tapped destination: \(destination.name)..")
```
Comment (johnnydhicks) [inline Resfeber/View Controllers/ItineraryViewController.swift:96]: And this one

Comment (DavidWrightOS) [reply Resfeber/View Controllers/ItineraryViewController.swift:96]: We will need this code in the future for presenting a detail view controller. In the meantime, I will replace the print statement with a `//TODO:` to help keep the codebase a little cleaner

File: Resfeber/View Controllers/FavoritesViewController.swift (hunk 1 above)
Comment (DavidWrightOS) [reply Resfeber/View Controllers/FavoritesViewController.swift:96]: We will need this code in the future for presenting a detail view controller. In the meantime, I will replace the print statement with a `//TODO:` to help keep the codebase a little cleaner

**Pull request merged**